SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -39,6 +39,7 @@ public static RequestFactory Configure(RequestFactory requestFactory)
                 .RegisterRequest<IGetRoslynExportProfileRequest, V5_20.GetRoslynExportProfileRequest>("5.2")
                 .RegisterRequest<IGetModulesRequest, V5_40.GetModulesRequest>("5.4")
                 .RegisterRequest<IGetRulesRequest, V5_50.GetRulesRequest>("5.5")
+                .RegisterRequest<IDownloadStaticFile, V5_50.DownloadStaticFile>("5.5")
```

Comment:
The SonarQube request handler is actually implemented in the SonarC# plugin, so I guess the min supported version is older, but we don't support SQ before 5.6 (LTS-1) anyway...